### PR TITLE
#1614 - Student File Upload - UI

### PR DESCRIPTION
**Pull request by ann-aot**
1. remove the formio button.
2. formio file uploader misalignment issue style fix  - actually it was happening for all files. formio file uploader.

image
image

### CODE REVIEW

File: sources/packages/forms/src/form-definitions/uploadstudentdocuments.json [hunk 1]
```diff
@@ -1,12 +1,13 @@
...
-  "display": "form",
-  "type": "form",
   "name": "uploadStudentDocuments",
   "path": "uploadstudentdocuments",
+  "type": "form",
+  "display": "form",
   "tags": [
     "common"
   ],
+  "owner": "63bf288c3e8304f36f5f1717",
```
Comment (andrewsignori-aot) [inline sources/packages/forms/src/form-definitions/uploadstudentdocuments.json:10]: We can get rid of the owner tag for this and every form.io definition that we updated.

**Review by andrewsignori-aot — APPROVED** [review]
Looks good 👍

File: sources/packages/forms/src/form-definitions/uploadstudentdocuments.json (hunk 1 above)
Comment (ann-aot) [reply sources/packages/forms/src/form-definitions/uploadstudentdocuments.json:10]: I will create a Pr after this to remove all other "owner"

**Review by andrepestana-aot — APPROVED** [review]
LGTM

**Review by guru-aot — APPROVED** [review]
LGTM, nice work @ann-aot

**Review by dheepak-aot — APPROVED** [review]
LGTM 👍

**Pull request merged**